File: src/main/kotlin/com/kapresoft/demo/FindMovieRequest.java
package com.kapresoft.demo;

public class FindMovieRequest {

    private String characterName;

    public String getCharacterName() {
        return characterName;
    }

    public void setCharacterName(String characterName) {
        this.characterName = characterName;
    }

    @Override
    public String toString() {
        final StringBuilder sb = new StringBuilder("FindMovieRequest{");
        sb.append("characterName='").append(characterName).append('\'');
        sb.append('}');
        return sb.toString();
    }
}

File: src/main/kotlin/com/kapresoft/demo/FindMovieRequest.kt
package com.kapresoft.demo

class FindMovieRequest {

    var characterName: String? = null

    override fun toString(): String {
        val sb = StringBuilder("FindMovieRequest{")
        sb.append("characterName='").append(characterName).append('\'')
        sb.append('}')
        return sb.toString()
    }

}